llama_stack/apis/datasetio/datasetio.py

```diff
@@ -37,8 +37,8 @@ class DatasetIO(Protocol):
     # keeping for aligning with inference/safety, but this is not used
     dataset_store: DatasetStore
 
-    @webmethod(route="/datasetio/rows", method="GET")
-    async def get_rows_paginated(
+    @webmethod(route="/datasets/{dataset_id}/iterrows", method="GET")
```

Comment:
`iter_rows`?

Reply:
maybe keep it `iterrows` to align with pandas?